Source/Utils/WarpXAlgorithmSelection.H

```diff
@@ -91,7 +91,35 @@ struct LoadBalanceCostsUpdateAlgo {
     };
 };
 
+/** Field boundary conditions at the domain boundary
+ */
+struct FieldBoundaryType {
+    enum {
+        PEC = 0,     //!< perfect electric conductor (PEC) with E_tangential=0
+        Periodic = 1,
+        PML = 2,
+        PMC = 3      //!< perfect magnetic conductor (PMC) with B_tangential=0
+    };
+};
+
+/** Particle boundary conditions at the domain boundary
+ */
+struct ParticleBoundaryType {
+    enum {
+        Absorbing = 0,     //!< particles crossing domain boundary are removed
+        Open = 1,          //!< particles cross domain boundary leave with damped j
```

Comment:
Are Open and Reflecting implemented yet?

Reply:
Not yet. I have a separate PR that will do this, #1728

Reply:
oh this is a good point. Thank you @atmyers 
absorbing is implemented in the code, but, it is not interfaced with the input parameters in this PR and therefore not mentioned in the documentation.
But currently we have` particles.boundary_conditions = absorbing`, where particles are discarded when they leave the domain.
I think open is the same as `particle_j_damping` for particles entering the pml.
These will be interfaced with the input in subsequent PRs.

Reply:
Ah, thanks!